appveyor.yml

```diff
@@ -3,7 +3,7 @@ environment:
  matrix:
   - DC: dmd
     DVersion: 2.089.0
-    arch: x64
+    arch: x86
```

Comment:
So that the MS tools (`cl.exe` and `lib.exe`) producing 32-bit outputs are used, in line with DMD defaulting to 32-bit code.